### PR TITLE
Add openapi to file example

### PR DESCRIPTION
**Pull request by ibek**

### CODE REVIEW

File: kafka-to-log/README.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ This example shows how to read a Kafka topic, using the camel `kafka` component,
 2. A topic named `test-topic` with some messages in it
 
 ## How to run
-1. Open the `kafka.camel.yaml` file and set the `brokers`, `topic` and `saslJaasConfig` values to match your Kafka cluster
```
Comment (lhein) [inline kafka-to-log/README.md]: I think this change (the whole file) should be separated out into a different PR as it is unrelated to the example you want to contribute. (separation of concerns)

**Comment by lhein** [comment]
Thank you for this example, Ivo!

File: kafka-to-log/README.md (hunk 1 above)
Comment (ibek) [reply kafka-to-log/README.md]: Ok, I removed that fix in this PR

Comment (lhein) [reply kafka-to-log/README.md]: thanks Ivo, appreciate your extra effort!

**Review by lordrip — APPROVED** [review]
Thanks a lot for the example and the fix @ibek, really appreciated 💪

**Review by lhein — APPROVED** [review]
